lib/Sabre/PhotosHome.php

```diff
@@ -95,7 +95,9 @@ public function getChild($name) {
 	}
 
 	/**
-	 * @return (AlbumsHome)[]
+	 * @return (AlbumsHome|PlacesHome|SharedAlbumsHome)[]
+	 *
+	 * @psalm-return array{0: AlbumsHome, 1: SharedAlbumsHome, 2: PlacesHome}
```

Comment:
Same, what is the difference ? Is `@psalm-return` allowing for a more fine grain type description ? If so, is `@return` still needed ?